docs/content/4.api/3.configuration.md

```diff
@@ -410,3 +410,10 @@ Toggles the document-driven mode.
 | `layoutFallbacks` |     `string[]`      | A list of `globals` key to use to find a layout fallback.      |
 | `injectPage`      |      `boolean`      | Inject `[...slug].vue` pre-configured page                     |
 | `trailingSlash`   |      `boolean`      | Add a slash at the end of  `canonical` and `og:url`            |
+
+## `respectPathCase`
+
+- Type: `Boolean`{lang=ts}
```

Comment:
```suggestion
- Type: `boolean`{lang=ts}
```
Currently making types in the docs lowercase. See #2167

Reply:
fixed..!